engine-rest/engine-rest-openapi/src/main/templates/paths/process-instance/variables-async/post.ftl

```diff
@@ -0,0 +1,74 @@
+{
+  <@lib.endpointInfo
+      id = "setVariablesAsyncOperation"
+      tag = "Process Instance"
+      desc = "Update or create runtime process variables in the root scope of process instances." />
```

Comment:
Should we also include the following hint from https://github.com/camunda/camunda-docs-manual/pull/664?

Please note that if `processInstanceIds`, `processInstanceQuery` and `historicProcessInstanceQuery` 
are defined, the resulting operation will be performed on the union of these sets.